tailcfg/tailcfg.go

```diff
@@ -223,6 +224,12 @@ type Hostinfo struct {
 	RoutableIPs   []wgcfg.CIDR `json:",omitempty"` // set of IP ranges this client can route
 	Services      []Service    `json:",omitempty"` // services advertised by this machine
 
+	// Network conditions:
+	MappingVariesByDestIP opt.Bool
+	WorkingIPv6           opt.Bool
+	WorkingUDP            opt.Bool
+	DERPLatency           map[string]int // keyed by hostname, in milliseconds
+
```

Comment:
A single number is probably not going to capture our latency analysis long-term. (We could use call-response keep-alives in DERP, and use those to measure RTT.) Maybe make the map value a struct and call this field "Average"?

Reply:
documented as "best recent time"